src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/PrisonerActivitiesCountRepository.kt

```diff
@@ -28,10 +29,9 @@ interface PrisonerActivitiesCountRepository : CrudRepository<PrisonerActivitiesC
           AND CS.SCHEDULE_DATE >= :startDate
           AND CS.SCHEDULE_DATE <= :endDate
         WHERE CA.AGY_LOC_ID = :agencyId 
-          AND OB.AGY_LOC_ID = :agencyId
```

Comment:
this restricted the prisoners to being still at the required prison, which the old code didn't do so stats were different.